xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -722,7 +729,8 @@
       <Docs>
         <param name="source">To be added.</param>
         <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Reads an Int16 out of a read-only span of bytes as big endian.
+            &lt;returns&gt;If the span is too small to contain an Int16, return false.&lt;/returns&gt;</summary>
```

Comment:
Fixing the formatting here but we also need to add the true and false conditions. Right now we're only explaining when you get false.

```suggestion
        <returns>If the span is too small to contain an <see cref="T:System.Int16" />, return <see langword="false" />.</returns>
```

Reply:
I just realized the returns was added inside the summary. I have to modify the file in my machine to extract it to the correct place.

Reply:
I fixed the internal returns.